VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-network-status/participant-network-status.component.scss

```diff
@@ -46,3 +46,34 @@
   height: auto;
   // margin-top: -1.8vw;
```

Comment:
remove this commented style if we don't need it

Reply:
removed the style